README.md

```diff
@@ -88,6 +88,23 @@ $profiler_data = $profiler->disable();
 $profiler->save($profiler_data);
 ```
 
+## Autoloader
+
+To be able to profile autoloader, this project provides `autoload.php` that
+loads classes needed to start up the profiler.
+
+Load it before loading composer autoloader:
+
+```php
+
+require_once '/path/to/autoload.php';
```

Comment:
This example could already be "vendor/perftools/php-profiler/autoload.php".
And the upcoming line (105) could also be "vendor/autoload.php" 
so that (lazy) people won't get problems when copy pasting things.

Reply:
1. users can insert the code anywhere in their project, the file is not likely in the project root, but `src` in their project.
2. includes without a path, can have an unwanted side effect. must use `__DIR__` to avoid lookup for a match from `include_path`.
2. I think it's good that people get errors if they copy paste and don't think :)

but I think it's good to indicate that `vendor/perftools/php-profiler/autoload.php` is in the file path, so they understand which autoload:
- vendor/perftools/php-profiler/autoload.php
- vendor/autoload.php

Reply:
there's also `examples/autoload.php`, sigh. probably should rename it to something different. currently I have no ideas.

Reply:
maybe `examples/auto_prepend_file.php`. actually I was against having it in repo anyway, as then have to maintain and handle various permutations of uses.

Reply:
Looks like a nice `multiple-handler.php` example to me. There are some other things too but nicely shows how to use the handler and handler-stack.

Reply:
it's a complete reference of config options, and stack handler is the recommended default.

- https://github.com/perftools/php-profiler/pull/44
- https://github.com/perftools/php-profiler/pull/35
- https://github.com/perftools/php-profiler/#stack-saver

Reply:
Actually, the Stack saver should be the default, so here's the PR:
- https://github.com/perftools/php-profiler/pull/57